### ISSUE TITLE
Activating 'Copy' from context menu minimizes main window

### ISSUE THREAD

**Issue by Moorelife** [notepad-plus-plus/notepad-plus-plus#4753]
### Description of the Issue
When a selected text is copied to the copy paste buffer using right-click and Copy from the context menu, the main window minimizes to just a small title bar with icons in the top left position on the screen. This does not happen when using ctrl-V, or when choosing Cut from the context menu.

### Steps to Reproduce the Issue

1. Open some text in Notepad++
2. Select a piece of text (a word in my case)
3. Right click the selection and choose Copy from the context menu.
4. Notice Notepad++ window being reduced to just title bar with buttons.

### Expected Behavior

Selected text should be copied, window should remain in current size and position.

### Actual Behavior

Selected text does get copied, but window is moved to 0,0 and minimized to just title bar buttons.

### Debug Information

Notepad++ v7.5.8   (32-bit)
Build time : Jul 23 2018 - 02:03:53
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll NppExport.dll 

![20180810094645](https://user-images.githubusercontent.com/11266191/43945692-3a168cb4-9c83-11e8-920e-a4e62e81df5e.png)

**Comment by ClaudiaFrank** [comment]
Does not happen to me - could you give the following a try
1. Stop npp
2. Temporarily rename the notepad++ directory under %APPDATA% (which normally expands to 
C:\Users\YOUR_USER_NAME\AppData\Roaming\Notepad++
3. Start npp

Do you still see the reported behavior?
If not, might you consider sharing the content of the temporarily renamed npp directory
by zipping it and upload to some hoster where one can download it?

**Comment by Moorelife** [comment]
Renaming the folder you mentioned does not alter the behavior:
Selecting text, right-clicking it, and choosing copy STILL 'minimizes' 
the Notepad++ window to the top left corner of my MAIN screen. (I have a three monitor setup)
See attached file for the requested folder (both renamed and newly made)
[Roaming.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/2293458/Roaming.zip)

**Comment by ClaudiaFrank** [comment]
Thx for testing - three monitor setup makes me asking whether you use Nvidia graphic cards and their
NView Manager (that was the name they used to call it some time ago - maybe nowadays it has a different name). The application which allows you to bind applications to a certain monitor etc...

If so, can you disable it for npp?

**Comment by Moorelife** [comment]
I have checked and the card is an NVidia K2200. Restricting the app for one program does not seem possible, but restricting the setup to one screen makes the problem disappear. As soon as I add one screen, the problem reappears again. I figure this is a very low-priority bug since it has a workaround by me using ctrl-C which I normally do, so don't waste too much time on it. If you need extra info, just let me know..... :-)

**Comment by ClaudiaFrank** [comment]
Actually I'm not sure if this is a npp bug at all as it has been reported that updating the NView Manager 
or the display driver might solve the issue.
And without using such cards, mainboard onboard graphics is sufficient for me, it is hard to debug at all.

**Comment by Moorelife** [comment]
Yup, no Notepad++ issue: it works fine on my two screen setup at home, but that is not nVidia....

Op 16 augustus 2018 15:15:07 +02:00, schreef ClaudiaFrank <notifications@github.com>:
